pkg/util/kernel/find_headers.go

```diff
@@ -85,6 +87,23 @@ func GetKernelHeaders(headerDirs []string, headerDownloadDir, aptConfigDir, yumR
 	}
 	log.Debugf("unable to find downloaded kernel headers: %s", err)
 
+	if strings.Contains(err.Error(), "linux/types.h missing") {
```

Comment:
Can we use a sentinel error for this, using `errors.New()` and checking for it using `errors.Is()`? Checking for strings is prone to be brittle.